pages/about.html

```diff
@@ -5,8 +5,24 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles.css" rel="stylesheet">
 </head>
 <body>
-  
+  <main class = "container">
+  <header>
+    <h1>Emily's Site</h1>
+    <nav class = "menu">
+      <a href = "index.html" id  = Home>Home</a>
```

Comment:
In html files, generally we don't put space around the equal sign used in attribute assignment, and we should always put quotes around the attribute values. And _usually_, ids wouldn't start with a capital letter (though this is more of a stylistic choice). So this line would look like

```html
      <a href="index.html" id="home">Home</a>
```